src/character.cpp

```diff
@@ -3680,9 +3680,9 @@ units::mass Character::weight_capacity() const
     return ret;
 }
 
-bool Character::can_pickVolume( const item &it, bool ) const
+bool Character::can_pickVolume( const item &it, bool, const item *avoid ) const
 {
-    if( weapon.can_contain( it ) ) {
+    if( weapon.can_contain( it ) && ( avoid == nullptr || &weapon != avoid )) {
```

Comment:
```suggestion
    if( weapon.can_contain( it ) && ( avoid == nullptr || &weapon != avoid ) ) {
```
I believe this is why the astyle check fails